okhttp/src/test/java/okhttp3/URLConnectionTest.java

```diff
@@ -685,6 +688,10 @@ private void connectViaHttpsReusingConnections(boolean rebuildClient) throws Exc
    * http://code.google.com/p/android/issues/detail?id=13178
    */
   @Test public void connectViaHttpsToUntrustedServer() throws Exception {
+    // https://github.com/square/okhttp/issues/5222
+    jdkMatchRule.expectFailure(platform.platformMatcher(PlatformRule.CONSCRYPT_PROPERTY),
```

Comment:
I find this syntax perplexing

Reply:
I like this simpler syntax you suggested.  Also I'll merge with PlatformTestRule

jdkMatchRule.assumeJdkLessThan(12)

Reply:
Have left a few helper methods, but cleaned it up to be externally understable I hope.